Firestore/core/src/firebase/firestore/local/local_serializer.cc

```diff
@@ -67,30 +70,29 @@ firestore_client_MaybeDocument LocalSerializer::EncodeMaybeDocument(
     case MaybeDocument::Type::Document:
       result.which_document_type = firestore_client_MaybeDocument_document_tag;
       result.document = EncodeDocument(static_cast<const Document&>(maybe_doc));
```

Comment:
Please add a TODO to port [this logic](https://github.com/firebase/firebase-ios-sdk/blob/master/Firestore/Source/Local/FSTLocalSerializer.mm#L88-L93) (checking whether the document contains a memoized proto).

Reply:
Added.